service/web/server/docs/openapi.yaml

```diff
@@ -0,0 +1,623 @@
+openapi: 3.0.3
+info:
+  title: BOPTEST Service API
+  description: A REST API for emulating buildings for testing control systems.
+  version: 0.9.0
+servers:
+  - url: http://localhost
+    description: Local server
+  - url: http://api.boptest.net
+    description: NREL BOPTEST Service
+components:
+  schemas:
+    BoundedSignalMeta:
+      type: object
+      properties:
+        Description:
+          type: string
+          description: Description of the signal.
+        Maximum:
+          type: number
+          nullable: true
+          description: Maximum value of the signal.
+        Minimum:
+          type: number
+          nullable: true
+          description: Minimum value of the signal.
+        Unit:
+          type: string
+          nullable: true
+          description: Unit of the signal.
+    SignalMeta:
+      type: object
+      properties:
+        Description:
+          type: string
+          description: Description of the signal.
+        Unit:
+          type: string
+          nullable: true
+          description: Unit of the signal.
+    BoundedSignalMetaResponse:
+      type: object
+      additionalProperties:
+        $ref: '#/components/schemas/BoundedSignalMeta'
+    SignalMetaResponse:
+      type: object
+      additionalProperties:
+        $ref: '#/components/schemas/SignalMeta'
+    ForecastQuery:
+      type: object
+      properties:
+        point_names:
+          type: array
+          items:
+            type: string
+          description: List of point names.
+        horizon:
+          type: number
+          description: Forecast horizon in seconds.
+        interval:
+          type: number
+          description: Time interval between forecast points in seconds.
+    ResultsQuery:
+      type: object
+      properties:
+        point_names:
+          type: array
+          items:
+            type: string
+          description: List of point names.
+        start_time:
+          type: number
+          description: Start time of the results query (epoch time).
+        final_time:
+          type: number
+          description: Final time of the results query (epoch time).
+    TimeSeries:
+      type: object
+      description: Time series response. Always includes a "time" array and any number of series arrays keyed by signal name.
+      properties:
+        time:
+          type: array
+          items:
+            type: number
+          description: Array of time points (epoch seconds).
```

Comment:
Time is not epoch seconds. It is simulation time with 0 as midnight on January 1. Correct the description.  Suggest "Array of time points (simulation time with 0 as midnight on January 1)".